src/gui/askforoauthlogindialog.cpp

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) Fabian Müller <fmueller@owncloud.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+
+#include "askforoauthlogindialog.h"
+#include "application.h"
+#include "creds/httpcredentialsgui.h"
+#include "theme.h"
+#include "ui_askforoauthlogindialog.h"
+
+#include <QClipboard>
+
+namespace OCC {
+
+AskForOAuthLoginDialog::AskForOAuthLoginDialog(AccountPtr accountPtr, QWidget *parent)
+    : QDialog(parent)
+    , _ui(new ::Ui::AskForOAuthLoginDialog)
+{
+    _ui->setupUi(this);
+
+    _ui->label->setText(tr("The account %1 has been logged out by the server.\n\nPlease use your browser to log into %2.").arg(accountPtr->displayName(), Theme::instance()->appNameGUI()));
```

Comment:
To "login to"

Reply:
I used `log into` everywhere else. I assume we have to adapt those occurrences as well then.

Reply:
Hmm yes  but we should discuss it in chat maybe we can find a native speaker